front_end/html/problem.html

```diff
@@ -225,7 +225,7 @@ <h2 style="display: inline">Expected output:</h2>
                 $.get("/get_submission/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}/" + submission_id, 
                     function (data, status) {
                         var json_data = JSON.parse(data);
-                        editor.setValue(json_data.code);
+                        editor.setValue(json_data.code,1);
```

Comment:
Super picky, but please put a space after the comma so that the code formatting is consistent.

Reply:
Actually, I was working on some other parts of the JavaScript and went ahead and implemented this in my version of the code (to avoid merge conflicts), so you can remove this change from your pull request. (But thanks for figuring out how to resolve the issue!)